### PR TITLE
[Merged by Bors] - Use humantime for http interval paramater

### PR DESCRIPTION
**Pull request by simlay**
This will allow for specifying any interval.

### CODE REVIEW

File: rust-connectors/sources/http/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Connector Change Log
 
-## http Version 0.2.1 - UNRELEASED
+## http Version 0.3.0 - UNRELEASED
+* `interval` field uses human readable values, "1s", "20ms", etc. ([PR #274](https://github.com/infinyon/fluvio-connectors/pull/274))
```
Comment (morenol) [inline rust-connectors/sources/http/CHANGELOG.md:4]: Also we can mention about linger time support

**Review by morenol — APPROVED** [review]
Just added a comment in the changelog

File: rust-connectors/sources/http/CHANGELOG.md (hunk 1 above)
Comment (simlay) [reply rust-connectors/sources/http/CHANGELOG.md:4]: That's an option shared amongst all the connectors. We need to figure out a good way to show all the common options.

**Pull request closed without merging**